packages/fast-storybook-presets/src/index.ts

```diff
@@ -0,0 +1,53 @@
+import path from "path";
+import glob from "glob";
+import { merge } from "lodash";
+
+/**
+ * Options object to configure the FAST DNA presets
+ */
+interface FASTDNAPresetOptions {
+    designSystemAddon: {
```

Comment:
nit: I feel like Addon should be AddOn

Reply:
This is the convention storybook is using, so I'm going to stick with this convention.